File: frontendbuddyfinv/src/models/ProductoEstrellaDTO.js
// models/ProductoEstrellaDTO.js
export class ProductoEstrellaDTO {
    constructor({ nombreProducto, cantidadVendida }) {
      this.nombreProducto = nombreProducto;   // Nombre del producto
      this.cantidadVendida = cantidadVendida; // Cantidad total vendida
    }
  }

File: backendbuddyfinv/src/main/java/com/es/backendbuddyfinv/dto/ProductoEstrellaDTO.java
package com.es.backendbuddyfinv.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ProductoEstrellaDTO {
    private String nombreProducto;   // Nombre del producto
    private Long cantidadVendida;    // Cantidad total vendida (SUM(dv.cantidad))
}